docs/components/List/List.stories.mdx

```diff
@@ -104,11 +107,14 @@ occurrence that has no section.
 ### **Custom Rendering (Advanced Usage)**
 
 If you need more control over the rendering of list items, you can provide a
-**custom render function** using the `customRenderItem` prop. This is useful if
-the default rendering of the `ListItem` component doesn’t meet your
-requirements. If a custom render function is provided onClick, url, and hover
-functionality will still work. Sectioned list functionality will also remain the
-same.
+**custom render function** using one or more of the `customRenderItem` or
+`customRenderSection` props.
+
+This is useful if the default rendering of the `ListItem` or section header
+component doesn’t meet your requirements. If a custom render function is
+provided onClick, url, and hover functionality will still work. Sectioned list
+functionality will also remain the same, only the appearance will change if a
```

Comment:
I would suggest being more explicit here and say "only the appearance of the header will change" since `customRenderSection` only changes the appearance of the header

Reply:
fair. ok pushed something, lemme know if that cleared it up!